### PR TITLE
#1109 Remove get table button from jdbc config form

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1109 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/pages/PipelinePage/Connectors/JdbcSource.js [hunk 1]
```diff
@@ -460,28 +398,15 @@ class JdbcSource extends React.Component {
...
-                  isWorking={isBtnWorking}
-                  disabled={isBtnWorking || isRunning}
-                  data-testid="get-tables-btn"
-                  handleClick={this.handleGetTables}
-                />
-              </TableWrapper>
+              <Input
+                name="table"
+                width="100%"
+                placeholder="column1"
```
Comment (eechih) [inline ohara-manager/client/src/components/pages/PipelinePage/Connectors/JdbcSource.js]: 是不是 table_name 比較恰當？

Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/Connectors/JdbcSource.js]: Good catch. Fixed :)

**Comment by wu87988622** [comment]
retry manager

**Comment by konekoya** [comment]
retry manager

**Comment by konekoya** [comment]
Failed build is unrelated

**Pull request merged**